dist/infragram.js

```diff
@@ -475,19 +475,19 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
           return pos = 0;
         }
       }
-
       function success(stream) {
         var video;
+        //localStream = this.stream;
```

Comment:
These too, just lower in the same file: https://github.com/publiclab/infragram/blob/3f5f87aefef84d92f87eed3e4b22a0cd6edc8bf0/src/io/camera.js#L79